webpack.prod.config.js

```diff
@@ -93,7 +93,7 @@ plugins: [
     }),
 
     new webpack.EnvironmentPlugin({
-      NETWORK: process.env.NETWORK,
+      NETWORK: "main",
```

Comment:
What if want to run a prod build against another network?  Can't do it with this change.

Reply:
that is the default value if process.env.NETWORK is not set, this is how you use EnvironmentPlugin